### ISSUE TITLE
Improve links to GitHub on "Applications" pages at www.kiwix.org

### ISSUE THREAD

**Issue by kelson42** [kiwix/web#229]
In many Kiwix presentation boxes at https://kiwix.org/en/applications/ we have a link to the corresponding GithHub repository. See
https://kiwix.org/en/applications/

This is right to put in emphasis:
* the free software nature of what we are doing
* the collaborative nature of its development

But I believe many things around this could/should be fixed:
* This is not available for all softwares.
* GitHub casing is wrong, it is written "Github" and it should be "GitHub".
* It's not always the exact same phrasing. I propose to stick everywhere to "Source code available on GitHub".
* I would propose to add afterward: "[Report issue](https://github.com/kiwix/kiwix-apple/issues)." to encourage people to do so!

**Comment by rgaudin** [comment]
I agree ; first point is the most important IMO. Looks like we have open and closed softwares there.

**Comment by rgaudin** [comment]
Still lacking macOS DMG and kiwix-tools for macOS.

Should we remove the link to kiwix-hotspot?

**Comment by sj80231** [comment]
Removing the link only creates confusion.  You need to scrape a lot of other information off the website so people are not expecting that you currently offer a functional hot-spot. The Kiwix website still invites people to purchase the hot-spot as part of your pre-configured packages.  I know, because I purchased one of those recently to get the hot-spot.  As you know though, there is no way to configure the hot-spot, even just to password protect it.  What's more, the links to content that are not functional, and the hot-spot causes certification errors (from IOS, and from Google) when I connect my iphone to it.  These errors pop up continually once you access the hot-spot, and continue for some time after disconnecting from the hot-spot.  I really thought your product was going to work, so: I bought a new RPi4B to be a dedicated host, a power supply for the Pi, a 1 TB SD card  Card (all in, over $150), plus, then I spent tons of time trying to make it work (that's probably the most wasteful part).  I opened an issue on GitHub, and received a couple replies, but no one said, "The hot-spot currently isn't working", "we'll refund your money".  Someone  could have saved me $150, and days of wasted time by doing more than just deleting a link to the configuration tool.  I hope there will be some resolution for all this in the future, as the idea behind this project is awesome.  In the mean time though, I hope I can save others from wasting their money and time because they are misled by what's advertised on the Kiwix site.

**Comment by sj80231** [comment]
I forgot to add in the cost of the "hot-spot" in the total above which brings me very close to a $200 investment in something that just shouldn't have been advertised in the first place.

**Comment by kelson42** [comment]
@sj80231 AFAIK your issue is currently treated at https://github.com/kiwix/kiwix-desktop/issues/1140 and at this stage it is unclear if there is an issue with SD image, your hardware or the way how you have flashed your SD card. Investigation is ongoing and answer to these questions will be found when it will be over. It's in any case better to not multiply the places where the discussion run.

**Comment by kelson42** [comment]
@rgaudin @Popolechien I will complete this issue myself so we can close it.

**Comment by x15sr71** [comment]
Hi @kelson42,

I noticed that the suggested improvements in issue #229 (related to GitHub links and phrasing on the Applications page) don’t seem to be reflected on the live website yet. I’d be happy to work on it and make a PR with the necessary changes.

However, after checking the repository mentioned in the issue, it doesn’t seem to match the live website content. Could you please confirm if there is a different repository where these changes should be made? I want to ensure I’m contributing to the right place.

Thanks for your assistance!

**Comment by kelson42** [comment]
@x15sr71 Hi, these fixes should be done by @Popolechien directly in Wordpress. There is nothing to fix in this repository.

**Comment by x15sr71** [comment]
@kelson42, Thanks for the clarification! I’m new to the Kiwix organization and would love to contribute. Could you guide me a bit on how I could get involved with fixing these issues in WordPress, or any other areas where I can help? I’m eager to contribute and learn more about your development process.

Thanks for your support!

**Comment by kelson42** [comment]
@x15sr71 I recomdend to read https://kiwix.org/code

**Comment by x15sr71** [comment]
@kelson42, Thanks for sharing the link! I’ll look at the code documentation on the Kiwix website to get a better understanding of the development process.

**Comment by rgaudin** [comment]
Covered
